src/main/java/net/sf/jabref/model/event/EntryChangedEvent.java

```diff
@@ -1,6 +1,5 @@
 package net.sf.jabref.model.event;
```

Comment:
I would actually sort the events in the corresponding package (i.e. model.entry or model.database, model.groups)